packages/bridge-ui/src/components/form/AddCustomERC20.svelte

```diff
@@ -0,0 +1,68 @@
+<script lang="ts">
+  import { ETH } from "../../domain/token";
+  import { Trash } from "svelte-heros-v2";
+  import { signer } from "../../store/signer";
+  import { token as tokenStore } from "../../store/token";
+  import { userTokens, userTokenStore } from "../../store/userTokenStore";
+  import Erc20 from "../icons/ERC20.svelte";
+  import Modal from "../modals/Modal.svelte";
+  import { LottiePlayer } from "@lottiefiles/svelte-lottie-player";
+
+  export let showAddressField = false;
+  export let addERC20;
+  export let loading = false;
+
+  let customTokens = [];
+  userTokens.subscribe(tokens => customTokens = tokens);
+
+  async function remove(token) {
+    const address = await $signer.getAddress();
+    const updatedTokensList = $userTokenStore.RemoveToken(token, address);
+    userTokens.set(updatedTokensList);
+    tokenStore.set(ETH);
+  }
+</script>
+
+<Modal title="Add custom ERC20" bind:isOpen={showAddressField}>
+  <form class="flex h-full min-h-tooltip-modal w-full flex-col justify-between" on:submit|preventDefault={addERC20}>
+    <input
+      type="text"
+      placeholder="Enter valid ERC20 Address"
```

Comment:
should this form not also show the symbol/decimals/etc when you paste the address in, like how Metamask works, to make sure you have the correct one? usually it is done with an on:input, like:
```ts
if(ethers.utils.isAddress(input)) {
  const contract = new ethers.Contract(input, ERC20_ABI, signer);
  const symbol = await contract.symbol();
  const decimals= await contract.decimals();
}
```